app/gateway/2.6.x/configure/auth/kong-manager/workspaces.md

```diff
@@ -73,21 +73,26 @@ navigate to the new Workspace's dashboard.
 
     ![New Dashboard](https://doc-assets.konghq.com/1.3/manager/workspaces/02-workspace-dashboard.png)
 
-
 ## Delete or Edit a Workspace
 
-To delete a Workspace, everything inside the
-Workspace must be deleted first. This includes default Roles on the "Admins"
-page.
+To be able to delete a workspace, *all data* must first be deleted from the workspace:
+
+* Via Kong Manager, turn off the Dev Portal. Go to Dev Portal **Settings** > **Advanced** > **Turn Off**.
+* Via Kong Manager, remove all roles. Go to **Teams** in the top navigation. Navigate to the **Roles** tab. Click **View** on the workspace you want to delete. Go to each entry and click **Edit**. In the entry detail page, click **Delete Role**. A confirmation modal will appear. Click **Delete Role** again.
+* Delete *all workspace files* using one of the following methods:
+  * Use the [Kong Portal CLI](https://github.com/Kong/kong-portal-cli), and run `portal wipe WORKSPACE_NAME`.
+  * Send a `DELETE` request to the Admin API to delete all files associated with the workspace. Construct your `DELETE` request to the following endpoint: [`/workspaces/{WORKSPACE_NAME}/files`](/gateway/{{page.kong_version}}/admin-api/workspaces/reference/#delete-a-workspace).
+  * Manually delete all files via Dev Portal > **Editor**. You cannot delete folders at this time, but deleting all files from a folder will remove the folder.
 
```

Comment:
Trying something out. This should look a lot cleaner in navtabs, and people should be used to this tool-based approach for completing a task since we do this a lot throughout the gateway docs:

> {% navtabs %}
> {% navtab Kong Manager %} 
> Using Kong Manager, complete the following:
> 
> 1.  Manually delete all files via Dev Portal > **Editor**. You cannot delete folders at this time, but deleting 
> all files from a folder will remove the folder.
> 1. Turn off the Dev Portal. Go to Dev Portal **Settings** > **Advanced** > **Turn Off**.
> 1. Remove all roles from the workspace:
>      1. Go to **Teams** in the top navigation.
>      1. Navigate to the **Roles** tab. 
>      1. Click **View** on the workspace you want to delete. 
>      1. Go to each role entry and click **Edit**. 
>      1. In the entry detail page, click **Delete Role**. A confirmation modal will appear. Click **Delete Role** again.
> 
> {% endnavtab %}
> {% navtab Admin API %}
>
> 1. Send a `DELETE` request to the Admin API to delete all files associated with the workspace. Construct your `DELETE` request to the following endpoint: [`/workspaces/{WORKSPACE_NAME}/files`](/gateway/{{page.kong_version}}/admin-api/workspaces/reference/#delete-a-workspace).
> 1. Turn off the Dev Portal for the workspace:
>    ```bash
>    curl -X PATCH http://localhost:8001/workspaces/{WORKSPACE_NAME}   --data "config.portal=true"
>    ```
> 1. Send a `DELETE` request to the Admin API to delete each role. Construct your `DELETE` request to the 
> following endpoint: [`/{workspace_name}/rbac/roles/{ROLE_NAME|ROLE_ID}`](/gateway/{{page.kong_version}}/admin-api/rbac/reference/#delete-a-role)
> 
> {% endnavtab %}
> {% endnavtabs %}



The only thing is that I have no idea where to put this option:

> Use the [Kong Portal CLI](https://github.com/Kong/kong-portal-cli), and run `portal wipe WORKSPACE_NAME`.

Maybe a third tab for "Portal CLI" ? There's also a way to disable the portal for the workspace through the CLI: (see https://github.com/Kong/kong-portal-cli). The only thing we can't do using this tool is deleting roles, but that step could be "switch to one of the other tabs and follow the instructions to remove all roles from the workspace".

Let me know if that all makes sense.

Reply:
The only thing about this is that each other requires turning off the Dev Portal and removing the roles as well. 

The only chunk that differs here specifically is how to delete the files.

Reply:
Actually - that's not entirely true. You can do all of these tasks through the Admin API (see my text above).

Reply:
Yeah, of the current content tho. So I can add more non-Kong Manager instructions